src/commands/local.mjs

```diff
@@ -0,0 +1,60 @@
+import { ensureContainerRunning } from "../lib/docker-containers.mjs";
+
+/**
+ * Starts the local Fauna container
+ * @param {import('yargs').Arguments} argv The arguments from yargs
+ * @returns {Promise<void>} a promise that resolves when the container is ready.
+ * It will reject if the container is not ready after the maximum number of attempts.
+ */
+async function startLocal(argv) {
+  await ensureContainerRunning({
+    imageName: argv.image,
+    containerName: argv.name,
+    hostPort: argv.hostPort,
+    containerPort: argv.containerPort,
+    pull: argv.pull,
+  });
+}
+
+/**
+ * Builds the yargs command for the local command
+ * @param {import('yargs').Argv} yargs The yargs instance
+ * @returns {import('yargs').Argv} The yargs instance
+ */
+function buildLocalCommand(yargs) {
+  return yargs.options({
+    containerPort: {
+      describe: "The port inside the container Fauna listens on",
+      type: "number",
+      default: "8443",
+    },
+    hostPort: {
+      describe:
+        "The port on the host machine mapped to the container's port. This is the port you'll connect to Fauna on.",
+      type: "number",
+      default: "8443",
+    },
+    image: {
+      describe: "The image to run locally",
+      type: "string",
+      default: "fauna/faunadb:latest",
+    },
```

Comment:
Why is this customizable?

Reply:
It probably doesn't need to be. I put it in case someone wanted to use the "nightly" container build. But taht seems like an internal concern not appropriate for this tooling.

Reply:
I removed this flag.